samples/net/wifi/overlay-psa.conf

```diff
@@ -0,0 +1 @@
+CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_MBEDTLS_PSA=y
```

Comment:
as this is only a single config, we can just remove the overlay and add a section in `wifi.rst` about how to enable PSA? (Independent of whether overlay is used or not, good to add a separate PSA section)